doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -307,8 +309,58 @@ Most cloud storage architectures incorporate a number of clustered storage nodes
+
+Availability of any single OpenStack cloud is dependent on a number of factors including: 
+-	environmental – dual connected power and PDUs, redundant cooling, rack distribution etc. 
+-	resilient network fabric – ToR (leaf), spine, overlay networking, underlay networking etc.   It is assumed that all network components are designed to be fault tolerant and all OpenStack controllers, computes and storage are dual-homed to alternate leaf switches.
+-	controller nodes setup in-line with the vendor recommendation (e.g. min 3 physical nodes)
+-	network nodes (where applicable) 
+-	storage nodes setup in line with the vendor recommendation
+-	compute nodes – enough to handle the workload after local failure scenario
+
+<a name="3.5.1"></a>
+### 3.5.1. Topology Overview
+
+Notes:
+- Region is represented by a single OpenStack control plane.
+- Resource Failure Domain is effectively the “blast radius” of any major infrastructure failure such as loss of PDU or network leafs.
+- Control planes includes redundant network nodes where OVS-kernel is used.
+- Controller nodes are typically a minimum of 3.
+- Shared storage is optional but it is important to ensure shared assets are distributed across serving clouds such as boot images.
+
+| Topology Ref| Type| Control Planes| Shared Storage (optional)| Compute AZs| Achievable Service Availability %| Service Multi-region awareness| Notes |
+|----------|-----|-----------------|---------------|-----------|---------|--------------|-------------|
+| 1 | Local Redundancy - workload spread across servers | 1 | 1 | 1 | Variable | Not required | Suitable where only limited local application availability is required e.g. nova anti-affinity |
+| 2 | Regional Redundancy - workload spread across AZs | 1 | >=2 | >=2 | >99.n | Not required | Suitable where local application HA is required. Control plane should be distributed across DC failure domains (assuming layer 2 connectivity) but may be unavailable during upgrades |
+| 3 | Global Redundancy - workload spread across multiple Regions | >=2 | >=2 | >=2 | >99.nn | Required e.g. GSLB | Suitable where local and region application HA is required Control plane could be kept available in one site during upgrades |
+
+<a name="3.5.2"></a>
+### 3.5.2. Topology Detail 
+
+#### 3.5.2.1. Topology 1	- Local Redundancy
+
+Under normal operation this deployment can handle a single failure of a controller node or storage node without any impact to the service.   If a compute node fails the application layer (often the VNFm) would need to restart workloads on a spare compute node of similar capability i.e. cloud may need to be provided with n+1 capacity.  In the case of an active/active application deployed to separate compute nodes (with hypervisor anti-affinity) then there would be no service impact.  
```

Comment:
"In the case of an active/active application deployed to separate compute nodes (with hypervisor anti-affinity) then there would be no service impact."

I think "then" is unnecessary

Reply:
Done - please review later once my change is committed